### ISSUE TITLE
node-id duplication

### ISSUE THREAD

**Issue by segevere** [IETF-OPSAWG-WG/l3nm#80]
why there is a need for two keys in node container?

       key "vpn-node-id ne-id";
        leaf vpn-node-id {
          type union {
            type l3vpn-svc:svc-id;
            type uint32;
          }
          description
            "Type STRING or NUMBER Serivice-Id";
        }
  
        leaf ne-id {
          type string;
          description
            "";
        }

**Comment by sbarguil** [comment]
I one of our operations, all the 2G/ 3G and 4G VRFs has the same ID in all the routers. Based on this:
```
vpn-id: "3G"
vpn-node-id="50013" 
```
We need to create several "vpn-nodes" in the same service. Thus, in this case the "ne-id" is the key to differentiate it.

**Comment by segevere** [comment]
but what is the difference in L3NM between vpn-node-id to ne-id?

**Comment by sbarguil** [comment]
vpn-node-id idetifies de vpn instance (i.e. VRF)
ne-id identifies the ne where the instance is created.

**Comment by oscargdd** [comment]
Agreement to leave only ne-id as unique key.

**Comment by oscargdd** [comment]
Closed with sha https://github.com/IETF-OPSAWG-WG/l3nm/commit/2db61d013664abd52411ab61e54958de054e03a6

**Comment by sbarguil** [comment]
Hello everybody,
I know we have discussed this topic previously, However, we have detected the following case:

Service-Name (vpn-id): Enterprise X
--
Ne-ID: 10.10.10.1
--
VRF1 (vpn-node-id): 100, Hub
VRF2 (vpn-node-id): 200, Spoke
 
We want to deploy two VPNs of the "Enterprise X" service in the same node. One acting as Hub and One as Spoke; In the same PE.
If we want to keep track of the relationship between the services the NE-ID key is not enough. Because we can not add two vpn-node-id part of the same service with the same NE-ID.

I suggest, to return to the previous two keys we have before.
